src/main/java/org/usfirst/frc4904/auton/Poses.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc4904.auton;
+
+import edu.wpi.first.wpilibj.geometry.Pose2d;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+
+class Poses {
+
+        //
+        // All measurements are in inches, starting at the bottom left corner of the
+        // field (I hope)
```

Comment:
BAD! Use everything in meters, and then use Units.inchesToMeters() if the measurements are by inches in default.